core/multivm_processor.go

```diff
@@ -0,0 +1,160 @@
+package core
+
+import (
+	"math/big"
+
+	"github.com/ethereumproject/sputnikvm-ffi/go/sputnikvm"
+	"github.com/ethereumproject/go-ethereum/common"
+	"github.com/ethereumproject/go-ethereum/core/state"
+	"github.com/ethereumproject/go-ethereum/core/types"
+	evm "github.com/ethereumproject/go-ethereum/core/vm"
+	"github.com/ethereumproject/go-ethereum/crypto"
+	"github.com/ethereumproject/go-ethereum/logger"
+	"github.com/ethereumproject/go-ethereum/logger/glog"
+)
+
+var UseSputnikVM = false
+
+func ApplyMultiVmTransaction(config *ChainConfig, bc *BlockChain, gp *GasPool, statedb *state.StateDB, header *types.Header, tx *types.Transaction, totalUsedGas *big.Int) (*types.Receipt, evm.Logs, *big.Int, error) {
```

Comment:
And now I feel like I'm just nitpicking... but exported vars and funcs should have comments, eg.
```
// ApplyMultiVmTransaction applies a transaction within SputnikVM...
func ApplyMultiVmTransaction(config ...
```